<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TouchHooksMainWindowUi</class>
 <widget class="QMainWindow" name="TouchHooksMainWindowUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TouchHooks2Tuio</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="plainText_guiTextArea">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_hooks">
    <property name="title">
     <string>Hooks</string>
    </property>
    <addaction name="action_useGlobalTouchHook"/>
   </widget>
   <widget class="QMenu" name="menuNetwork">
    <property name="title">
     <string>Network</string>
    </property>
    <addaction name="action_tuioUdpChannelOne"/>
    <addaction name="action_tuioUdpChannelTwo"/>
    <addaction name="action_flashXmlTcpChannel"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_hooks"/>
   <addaction name="menuNetwork"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_quit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_removeGlobalTouchHook">
   <property name="text">
    <string>Remove Global Touch Hook</string>
   </property>
  </action>
  <action name="action_useGlobalTouchHook">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use Global Touch Hook</string>
   </property>
  </action>
  <action name="action_attachTouchHookByPid">
   <property name="text">
    <string>Attach Touch Hook by PID</string>
   </property>
  </action>
  <action name="action_tuioUdpChannelOne">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>TUIO UDP Channel 1</string>
   </property>
  </action>
  <action name="action_tuioUdpChannelTwo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>TUIO UDP Channel 2</string>
   </property>
  </action>
  <action name="action_flashXmlTcpChannel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Flash XML Channel</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
